web/themes/custom/usagov/templates/wizard_2024/html--taxonomy--term--wizard.html.twig

```diff
@@ -0,0 +1,117 @@
+{# do some weird stuff with the indentation here to make it appear correctly in the DOM #}
+{% for link in breadcrumb_block['content']['#links'] %}
+{% if loop.first %}
+    {{ 'Taxonomy_Text_1'|json_encode(constant('JSON_PRETTY_PRINT'))|raw }}: {{ link.text|json_encode(constant('JSON_PRETTY_PRINT'))|raw }},
+    {% if page_type == 'wizard-start' %}
+    {% set titleCounter = titleCounter + 1 %}
+    {% endif %}
+{% elseif loop.index == 2 %}
+{{ 'Taxonomy_Text_2'|json_encode(constant('JSON_PRETTY_PRINT'))|raw }}: {{ wizardTitle|json_encode(constant('JSON_UNESCAPED_SLASHES'))|raw }},
+    {% if page_type != 'wizard-start' %}
+{{ 'Taxonomy_Text_3'|json_encode(constant('JSON_PRETTY_PRINT'))|raw }}: {{ link.text|json_encode(constant('JSON_UNESCAPED_SLASHES'))|raw }},
+    {% endif %}
+{% elseif loop.index <= 5 %}
+{% set textTag = 'Taxonomy_Text_' ~ titleCounter %}
+{% set titleCounter = titleCounter + 1 %}
+{{ textTag|json_encode(constant('JSON_PRETTY_PRINT'))|raw }}: {{ link.text|json_encode(constant('JSON_UNESCAPED_SLASHES'))|raw }},
+{% endif %}
+{% endfor %}
+{% for i in range(1, 6) %}
+{% if titleCounter <= 6 %}
+{% set lastText = 'Taxonomy_Text_' ~ titleCounter %}
+    {{ lastText|json_encode(constant('JSON_PRETTY_PRINT'))|raw }}: {{ name|json_encode(constant('JSON_PRETTY_PRINT'))|raw }},
+{% endif %}
+{% set titleCounter = titleCounter + 1 %}
+{% endfor %}
+{% for link in breadcrumb_block['content']['#links'] %}
+{% if loop.first %}
+    {{ 'Taxonomy_URL_1'|json_encode(constant('JSON_PRETTY_PRINT'))|raw }}: {{ link.url.toString|json_encode(constant('JSON_UNESCAPED_SLASHES'))|raw }},
+{% if page_type == 'wizard-start' %}
+{% set urlCounter = urlCounter + 1 %}
+{% endif %}
+{% elseif loop.index == 2 %}
+    {{ 'Taxonomy_URL_2'|json_encode(constant('JSON_PRETTY_PRINT'))|raw }}: {{ wizardTitleUrl|json_encode(constant('JSON_UNESCAPED_SLASHES'))|raw }},
+    {% if wizardTitleUrl != link.url.toString %}
+{{ 'Taxonomy_URL_3'|json_encode(constant('JSON_PRETTY_PRINT'))|raw }}: {{ link.url.toString|json_encode(constant('JSON_UNESCAPED_SLASHES'))|raw }},
+    {% endif %}
+{% elseif loop.index <= 5 %}
+{% set urlTag = 'Taxonomy_URL_' ~ urlCounter %}
+{% set urlCounter = urlCounter + 1 %}
+    {% if page_type == 'wizard-start' %}
+{{ urlTag|json_encode(constant('JSON_PRETTY_PRINT'))|raw }}: {{ currentPath|json_encode(constant('JSON_UNESCAPED_SLASHES'))|raw }},
+    {% else %}
+{{ urlTag|json_encode(constant('JSON_PRETTY_PRINT'))|raw }}: {{ link.url.toString|json_encode(constant('JSON_UNESCAPED_SLASHES'))|raw }},
+    {% endif %}
+{% endif %}
+{% endfor %}
+{% for i in range(1, 6) %}
+{% if urlCounter <= 6 %}
+{% set lastUrlCounter = 'Taxonomy_URL_' ~ urlCounter %}
+  {{ lastUrlCounter|json_encode(constant('JSON_PRETTY_PRINT'))|raw }}: {{ currentPath|json_encode(constant('JSON_UNESCAPED_SLASHES'))|raw }}{% if urlCounter < 6 %},
+  {% endif %}
+{% endif %}
+{% set urlCounter = urlCounter + 1 %}
+{% endfor %}
+    %}];
```

Comment:
Here's another % that doesn't belong. It's causing output like this: 
```
"Taxonomy_URL_6": "/scam-reporting"    %}];
```

Reply:
Hi @akf, thank you for catching these. Need to look into how that happened. I've pushed up the changes.